packages/requester-node-http/src/createNodeHttpRequester.ts

```diff
@@ -12,14 +12,17 @@ export type NodeHttpRequesterOptions = {
   httpsAgent?: https.Agent;
 };
 
+const agentOptions = { keepAlive: true };
```

Comment:
this can be done inline now, I think it makes more sense

Reply:
well, it's just to be sure both agent are in sync, don't you think it's less error prone like that?